superset-frontend/.prettierignore

```diff
@@ -26,3 +26,4 @@ CHANGELOG.md
 *-topo.json
 temporary_superset_ui/
 storybook-static/
+plugins/legacy-plugins-chart-histogram/src/Histogram.jsx
```

Comment:
why is this necessary. Can you just run prettier or fix the error instead of ignoring this file?

Reply:
When I first ran Prettier on the file, it didn't change anything. I suspect it was because either I misconfigured Prettier on my local machine, or Prettier confused and couldn't reformat. I will try to configure it again.